components/portfolio.js

```diff
@@ -0,0 +1,140 @@
+                            </p>
+                          </div>
+                        </div>
+                      </a>
+                      </Link>
+                    </div>
+                  </div>
+                  
+                  <div className="col-lg-3 col-md-3 col-sm-6 col-xs-12 mix dev biz">
+                    <div className="item">
+                    <Link href="/portfolio/sme">
+                      <a
+                         
+                      >
+                        <img
+                          src="assets/img/work/p2.png"
+                          alt="SME is an accounting, inventory, procurement, and payroll system specifically tailored for Philippine small and medium-scale businesses."
+                        />
+                        <div className="overlay">
+                          <div className="overlay-text">
+                            <h3>SME</h3>
+                            <p>
+                              SME is an accounting, inventory, procurement, and
+                              payroll system specifically tailored for Philippine
+                              small and medium-scale businesses.
+                            </p>
+                          </div>
+                        </div>
+                      </a>
+                      </Link>
+                    </div>
+                  </div>
+                  <div className="col-lg-3 col-md-3 col-sm-6 col-xs-12 mix illustrate typography web biz">
+                    <div className="item">
+                    <Link href="/portfolio/doorbell">
+                      <a
+                        
+                      >
+                        <img src="assets/img/work/p3.png" alt='' />
+                        <div className="overlay">
+                          <div className="overlay-text">
+                            <h3>Doorbell</h3>
+                            <p>
+                              Doorbell is an app built for real estate agents and
+                              investors. Doorbell allows you to be notified via SMS
+                              or email if there is a foreclosed property in the
+                              market that suits your needs.
+                            </p>
+                          </div>
+                        </div>
+                      </a>
+                      </Link>
+                    </div>
+                  </div>
+
+                  <div className="col-lg-3 col-md-3 col-sm-6 col-xs-12 mix illustrate typography web biz">
+                    <div className="item">
+                    <Link href="/portfolio/sarisuki">
+                      <a
+                       
+                      >
+                        <img src="assets/img/work/p4.png" alt='' />
+                        <div className="overlay">
+                          <div className="overlay-text">
+                            <h3>Doorbell</h3>
```

Comment:
Should this be for Sarisuki?

```suggestion
                            <h3>Sarisuki</h3>
```